filament/backend/src/opengl/PlatformCocoaGL.mm

```diff
@@ -115,7 +115,7 @@
                     queue: nil
                     usingBlock: notificationHandler];
                 [strongObservers addObject: observer];
-                
+
```

Comment:
Please revert unrelated whitespace changes, these surely weren't introduced by us, only going to cause conflicts in the future.